app/bingey-api/models/title.js

```diff
@@ -0,0 +1,42 @@
+const { mongoose, Schema } = require('mongoose');
+
+const titleSchema = new Schema(
+  {
+    _id: {

```

Comment:
There is no need to specify this, mongodb generates this property automatically

Reply:
Oh i forgot we will be filling these from imdb dataset. You should add _id: false along with timestamps so a secondary id is not generated

Reply:
Added `_id: false` to schema options